### DISCUSSION TITLE
is this solution better than linkwarden which is similar solution

### DISCUSSION THREAD

**Discussion by Greatz08** [omnivore-app/omnivore#3717] [General]
I am using link warden to create book mark or archive pages or and read things.Pretty awesome foss solution which can be selfhosted and accessed easily with url and beautiful interface so my question is that is this solution better than linkwarden and if yes then how ?

**Comment by jacksonh** [comment]
I've never used link warden so not sure I can answer. You might have some luck getting answers in Discord but probably wont get any responses here so I'm going to close this one. Feel free to jump onto Discord and ask
